src/shellCommands/ocSdkCommands.ts

```diff
@@ -375,3 +406,13 @@ async function getRequest(apiUrl: string): Promise<string | undefined> {
 function getLatestCollectionVersion(jsonData: any): string | undefined {
   return jsonData?.data?.collection?.latest_version?.version ?? jsonData?.data[0]?.version;
 }
+
+function getFinalPlaybookTaskFailure(stdOutput: string): string | undefined {
+  const playbookTasksFailureRegex = /FAILED! => {.*}/g;
+  const playbookTasksFailures = stdOutput.match(playbookTasksFailureRegex);
+  const finalFailureObject = playbookTasksFailures?.[playbookTasksFailures.length - 1]?.replace("FAILED! => ", "");
+  if (finalFailureObject) {
+    return JSON.parse(finalFailureObject)?.["msg"];
```

Comment:
That's true, sometimes the msg comes straight from playbook and is probably more informative but other times its just something like `non-zero return code`. Maybe we can check if there is an msg, that isn't just the default and display it but if there isn't display `stderr_lines` (because `stderr_lines`?

Reply:
I had this doubt too, based on [the return values documentation](https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html#stderr-lines), they say that `stderr-lines` is "always provided" and the `msg` is more generic, but we can take that path too